tool/grind.dart

```diff
@@ -268,7 +269,7 @@ coverage() {
 }
 
 @DefaultTask()
-@Depends(analyze, testCli, testWeb, coverage, build)
+@Depends(generateProtos, analyze, testCli, testWeb, coverage, build)
```

Comment:
I would consider making as separate step so that `grind deploy` only deploys what's committed to source control

Reply:
The proto generated classes have to be part of the deployed source base. What are you trying to hold back? 

/me is confused

Reply:
```suggestion
@Depends(analyze, testCli, testWeb, coverage, build)
```